addons/website/static/src/snippets/s_popup/000.js

```diff
@@ -74,6 +77,8 @@ const PopupWidget = publicWidget.Widget.extend({
      * @private
      */
     _onCloseClick: function () {
+        const nbDays = this.$target.find('.s_popup_main').data('consentsDelay');
```

Comment:
Sorry to annoy you with that but could you use `consentsDuration` instead ? Delay would mean "le temps que tu as pour consentir" to me :wink: